src/main/java/makaroshyna/onlinebookstore/dto/cartitem/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package makaroshyna.onlinebookstore.dto.cartitem;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class CreateCartItemRequestDto {
+    @NotNull
+    private Long bookId;
+    @Positive
```

Comment:
what if user sends null? and is 0 an actually valid case? I think we should have min value 1 because if quantity is 0 that means we want to remove that item through another endpoint

Reply:
Quantity is int so it can't be null or zero with this annotation. As you can see this annotation works for me :)
![image](https://github.com/makaroshyna/online-book-store/assets/83081198/66cf1c7c-e132-41ad-bf6f-e89b80ffdcee)
![image](https://github.com/makaroshyna/online-book-store/assets/83081198/9d12a61f-ee72-42a0-a69b-e544dcf2c854)
![image](https://github.com/makaroshyna/online-book-store/assets/83081198/346499b4-45b3-48d7-a2bf-749bdc128a84)
![image](https://github.com/makaroshyna/online-book-store/assets/83081198/9eb3b87e-0976-4c8d-a531-92441a84b75a)

Reply:
Maybe it's because int can't be null? I saw that null are valid too, but somehow it doesn't validate them here.